.github/workflows/python-package.yml

```diff
@@ -60,7 +60,15 @@ jobs:
       run: |
         sudo apt update
         sudo apt -y install maven samtools bcftools parallel libbz2-dev liblzma-dev
+    - name: Set up ascp binary
+    # Might need to periodically update this URL
+      run: |
+        curl https://d3gcli72yxqn2z.cloudfront.net/connect_latest/v4/bin/ibm-aspera-connect_4.1.3.93_linux.tar.gz -o ibm-aspera-connect.tar.gz
```

Comment:
This seems like a randomly generated URL. Is this guaranteed to be around for a while?

Reply:
Guarantee is a prety string word. I can say that it seems stable over couple of days but it might indeed change. I have not found a better way to retrieve Aspera connect though. I'm hoping they will be stable enough or have redirect if it moves.